spec/forms/hyrax/forms/permission_template_form_spec.rb

```diff
@@ -195,12 +195,14 @@
 
     context "when a new workflow has been chosen" do
       it "gives the managers workflow roles" do
+        expect(form).not_to receive(:update_permission_template)
```

Comment:
I don't see these as being particularly useful assertions.  There are an infinite variety of messages it is not receiving, why pick this one?